src/modules/common/MediaDisplay.tsx

```diff
@@ -183,10 +183,19 @@ export function MediaDisplay({ media }: { media: IMedia }) {
   const [opened, setOpened] = useState(false);
   const { secondaryColor } = useColorTheme();
   const { t } = useTranslation('modules/common-media-cards');
+  const { config } = useConfigContext();
 
   return (
     <Group noWrap style={{ maxHeight: 250, maxWidth: 400 }} p={0} m={0} spacing="xs">
-      <Image src={media.poster} height={200} width={150} radius="md" fit="cover" />
+      <a
+        href={media.overseerrId ?? 'javascript:'}
```

Comment:
what? ID or javascript: ? Can you explain this or rename the variable, if it contains an URI

Reply:
You need to add comments to your code and please use <Anchor /> from mantine when using links.